ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_contraindicatie/fhir_instance/mci-MCI-zwangerschap-2.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flag xmlns="http://hl7.org/fhir">
+   <id value="mci-MCI-zwangerschap-2"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/cio-MedicationContraIndication"/>
+   </meta>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationContraIndication.RegistrationDateTime">
+      <valueDateTime value="${DATE, T, Y, -29}T10:11:42+02:00"/>
+   </extension>
+   <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationContraIndication.RelationContraIndication">
+      <valueReference>
+         <reference value="Flag/mci-MCI-zwangerschap-1"/>
+         <type value="Flag"/>
+         <display value="Zwangerschap"/>
+      </valueReference>
+   </extension>
+   <identifier>
+      <system value="urn:oid:2.16.840.1.113883.2.4.3.11.999.26.1.2"/>
+      <value value="MCI-zwangerschap-2"/>
+   </identifier>
+   <status value="inactive"/>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="140401000146105"/>
+         <display value="contra-indicatie met betrekking op medicatiebewaking"/>
+      </coding>
+   </category>
+   <code>
+      <coding>
+         <system value="urn:oid:2.16.840.1.113883.2.4.4.1.902.40"/>
+         <code value="1320"/>
+         <display value="Zwangerschap"/>
+      </coding>
+   </code>
+   <subject>
+      <reference value="Patient/patient-XXX-Roelfsema"/>
+      <type value="Patient"/>
+      <display value="Patient, N XXX_Roelfsema"/>
+   </subject>
+   <period>
+      <start value="${DATE, T, Y, -30}T08:54:38+02:00"/>
```

Comment:
beetje rare data ,niet? Hoezo is iemand 12 maanden zwanger?

Reply:
Helemaal mee eens; ik zet dit op de backlog voor de IA'ers om in een volgende versie te verbeteren!